syz-manager/manager.go

```diff
@@ -755,8 +754,8 @@ func (mgr *Manager) runInstance(index int) (*Crash, error) {
 	return crash, nil
 }
 
-func (mgr *Manager) runInstanceInner(index int, instanceName string, injectLog <-chan []byte) (
-	*report.Report, []byte, error) {
+func (mgr *Manager) runInstanceInner(index int, instanceName string, injectLog <-chan []byte,
+) (*report.Report, []byte, error) {
```

Comment:
Was this an intentional change? Surprised this compiles :)

Reply:
Reverted the change.